spec/models/message_spec.rb

```diff
@@ -124,4 +124,23 @@
       expect(claim.state).to eq 'part_authorised'
     end
   end
+
+  describe '.s3_headers' do
```

Comment:
**not a blocker**, but be good to have a shared example to use in the other models (document_attachment and stats_report)

Reply:
Noted, I'll work on that now